docs/crio.conf.5.md

```diff
@@ -280,12 +280,6 @@ CRI-O reads its configured registries defaults from the system wide containers-r
 **image_volumes**="mkdir"
   Controls how image volumes are handled. The valid values are mkdir, bind and ignore; the latter will ignore volumes entirely.
 
-**insecure_registries**=[]
-  List of registries to skip TLS verification for pulling images.
```

Comment:
Is this worth it? While I’m all for removing as much of the search mechanism as we can get away with, this just forces users to migrate an option from one file to another with not much to gain.

Reply:
I was under the impression that was the consensus yesterday.  Previously CRI-O did not even consult the registries.conf for insecure registries which drove me (even more) to remove it.

But I am completely open to revert that part.

Reply:
`c/image/docker.{newDockerClient,newImageSourceAttempt}` apply the `registries.conf` configuration automatically, and that should affect CRI-O as well already without CRI-O having any explicit code to do so.

(Note the other comments, the semantics is a bit different.)

Reply:
Now that the `insecure_registries` part has been dropped, the release note in the PR description needs updating.

Reply:
Cheers!